cms/src/test/java/uk/nhs/digital/apispecs/module/ApiSpecSyncFromApigeeModuleTest.java

```diff
@@ -23,16 +24,20 @@
 import org.onehippo.repository.scheduling.RepositoryScheduler;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
+import uk.nhs.digital.apispecs.jobs.ApiSpecRerenderJob;
 import uk.nhs.digital.apispecs.jobs.ApiSpecSyncFromApigeeJob;
 
+import java.io.InputStream;
+import java.util.List;
 import javax.jcr.Node;
 import javax.jcr.RepositoryException;
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({HippoServiceRegistry.class, ApiSpecSyncFromApigeeModule.class})
 public class ApiSpecSyncFromApigeeModuleTest {
 
-    public static final String JOB_NAME = "apiSpecSyncFromApigee";
+    public static final String DAILY_JOB_NAME = "apiSpecSyncFromApigee";
+    public static final String NIGHTLY_JOB_NAME = "apiSpecRefresh";
```

Comment:
`apiSpecRefresh` wants to be `apiSpecRerender` as it breaks the test, otherwise.